cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -34,6 +34,7 @@ type MasterConfiguration struct {
 	CloudProvider      string     `json:"cloudProvider"`
 	NodeName           string     `json:"nodeName"`
 	AuthorizationModes []string   `json:"authorizationModes,omitempty"`
+	Rootfs             string     `json:"rootfs"`
```

Comment:
do we really need to expose it here ?

Reply:
Oh I see, I didn't realise how this differed from the other `../types.go`.  No, we don't need it exposed in this (persisted) version.